src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+function stopDefAction(evt) {
+  evt.preventDefault();
+}
+
+document.getElementById('button-send').addEventListener(
+  'click', stopDefAction, false
+);
```

Comment:
Давай підправимо на ось такий код, бо при такому підході взагалі не валідується форма.

```suggestion
function stopDefAction(evt) {
  evt.preventDefault();
}

document.addEventListener('submit', stopDefAction);
```